olive/model/handler/mixin/hf_config.py

```diff
@@ -44,6 +50,63 @@ def get_hf_model_config(self):
 
         return get_hf_model_config(self._get_model_path_or_name(), **self.hf_config.get_loading_args_from_pretrained())
 
+    def save_hf_model_config(self, output_dir: str, **kwargs):
```

Comment:
it seems those new added method is only used in the mixin class? if that's the case, we could make them private by adding prefix "_" to make it clear.

Reply:
All save_XXX functions removed in follow up PR.